packages/conversation/src/rest/v1/conversation-domain-api.ts

```diff
@@ -86,13 +87,14 @@ export class ConversationDomainApi implements Api {
     return this.client;
   }
 
-  private buildHostname(region: ConversationRegion) {
+  private buildHostname(region: ConversationRegionFlexible) {
+    const formattedRegion = region === '' ? region : `${region}.`;
```

Comment:
I think it come from myself, but thank you to small function usage here: my first reading of this line was cause of reflexion to understand logic and thinking there was a issue:
- testing against an empty value to use the testing value (the empty one)
- if not empty: use it too....

🤔 
Then I realized the final dot for second case; then use to format a string
Just a personal thought I think I would have written logic as 
```const formattedRegion = region !== '' ? `${region}.` '';```
Which could highlight in a better way the logic and use case.
Again just a personal thought and not a blocker

Reply:
You are right, it would be more readable.